src/Microsoft.NetCore.Analyzers/Core/MicrosoftNetCoreAnalyzersResources.resx

```diff
@@ -1074,4 +1074,13 @@
   <data name="JsonNetMaybeInsecureSerializerTitle" xml:space="preserve">
     <value>Ensure that JsonSerializer has a secure configuration when deserializing</value>
   </data>
+  <data name="SetHttpOnlyForHttpCookie" xml:space="preserve">
+    <value>Set HttpOnly to true for HttpCookie</value>
+  </data>
+  <data name="SetHttpOnlyForHttpCookieDescription" xml:space="preserve">
+    <value>Websites that use cookies must specify HttpOnly for all cookies containing sensitive information that are not explicitly required by legitimate scripts in the webpage. Eg. Of sensitive cookies are those used in authentication, authorization, session management or that otherwise contain PII.</value>
+  </data>
+  <data name="SetHttpOnlyForHttpCookieMessage" xml:space="preserve">
+    <value>HttpCookie.HttpOnly is set to false or not set at all when return HttpCookie or invalid type of cookie is created</value>
```

Comment:
>HttpCookie.HttpOnly is set to false or not set at all when return HttpCookie or invalid type of cookie is created [](start = 11, length = 113)

HttpCookie.HttpOnly is set to false or not set at all when using an HttpCookie.  Ensure security sensitive cookies are marked as HttpOnly to prevent malicious scripts from stealing the cookies.